Makefile

```diff
@@ -70,6 +69,16 @@ endif
 TARGET_ARCH = $(HOST_ARCH)
 
 # ---------------------------------------------------------------------
+# Code signing
+# ---------------------------------------------------------------------
+
+ifeq ($(TARGET_PLATFORM),darwin)
+ifndef CODESIGN_IDENTITY
+$(warning No code-sign identity found (CODESIGN_IDENTITY))
```

Comment:
Should this warning end with something like `(\$CODESIGN_IDENTITY isn't set)`. I'm not sure how useful `(CODESIGN_IDENTITY)` is supposed to be? ;)

P.S. Just noticed that CODESIGN can either be read as "code-sign" or "co-design" ;-)